Software/lib/MX25R1635/MX25R16.c

```diff
@@ -119,93 +113,28 @@ int MX25R16_Read(MxChip *Mxic, u32 Addr, u32 ByteCount, u8 *Buf)
+ *                Ths function invokes MxPP() when used with Mx25R16 external flash
  */
 int MX25R16_Write(MxChip *Mxic, u32 Addr, u32 ByteCount, u8 *Buf)
 {
     return Mxic->AppGrp._Write(Mxic, Addr, ByteCount, Buf);
 }
 
-/*
- * Function:      MXR2516_BufferRead
- * Arguments:      Mxic:      pointer to an mxchip structure of nor flash device.
- *                Addr:      device address to read.
- *                ByteCount: number of bytes to read.
- *                Buf:       pointer to a data buffer where the read data will be stored.
- * Return Value:  MXST_SUCCESS.
- *                MXST_FAILURE.
- * Description:   This function is for read the page buffer.
- */
-int MXR2516_BufferRead(MxChip *Mxic, u32 Addr, u32 ByteCount, u8 *Buf)
-{
-    return MxRDBUF(Mxic, Addr, ByteCount, Buf);
-}
-
-/*
- * Function:      MXR2516_BufferWrite
- * Arguments:      Mxic:      pointer to an mxchip structure of nor flash device.
- *                Addr:      device address to write.
- *                ByteCount: number of bytes to write.
- *                Buf:       pointer to a data buffer where the write data will be stored.
- * Return Value:  MXST_SUCCESS.
- *                MXST_FAILURE.
- * Description:   This function is for write the page buffer to implement RWW function.
- */
-int MXR2516_BufferWrite(MxChip *Mxic, u32 Addr, u32 ByteCount, u8 *Buf)
-{
-    int Status;
-    Status = MxIsFlashBusy(Mxic);
-    if (Status == MXST_DEVICE_BUSY)
-        return Status;
-
-    if(Addr <= 0x0FFFFFF)
-        Mxic->BankStatus = BANK0_BUSY;
-    else if(Addr <= 0x1FFFFFF)
-        Mxic->BankStatus = BANK1_BUSY;
-    else if(Addr <= 0x2FFFFFF)
-        Mxic->BankStatus = BANK2_BUSY;
-    else
-        Mxic->BankStatus = BANK3_BUSY;
-
-    if(Mxic->WriteBuffStart == FALSE)
-    {
-        Mxic->WriteBuffStart = TRUE;
-        Status = MxWRBI(Mxic, Addr, ByteCount, Buf);
-        if (Status != MXST_SUCCESS)
-            return Status;
-    }
-    else if(ByteCount > 0)
-    {
-        Status = MxWRCT(Mxic, Addr, ByteCount, Buf);
-        if (Status != MXST_SUCCESS)
-            return Status;
-    }
-
-    if(ByteCount == 0)
-    {
-        Status = MxWRCF(Mxic);
-        if (Status != MXST_SUCCESS)
-            return Status;
-        Mxic->WriteBuffStart = FALSE;
-    }
-    return MXST_SUCCESS;
-}
-
 /*
  * Function:      MX25R16_Erase
  * Arguments:      Mxic:           pointer to an mxchip structure of nor flash device.
  *                Addr:           device address to erase.
- *                EraseSizeCount: number of block or sector to erase.
+ *                EraseSizeCount: number of blocks to erase.
  * Return Value:  MXST_SUCCESS.
  *                MXST_FAILURE.
  *                MXST_TIMEOUT.
  * Description:   This function erases the data in the specified Block or Sector.
  *                 Function issues all required commands and polls for completion.
```

Comment:
Spacing

Reply:
What is the issue here?

Reply:
Odd spacing, nothing big. You can leave it if you want, the only change I wanted was the misspelled this

Reply:
Ok done